x-pack/plugins/security_solution/public/detections/pages/detection_engine/rules/create/helpers.test.ts

```diff
@@ -112,8 +112,8 @@ describe('helpers', () => {
     });
 
     test('returns formatted object as DefineStepRuleJson', () => {
-      const result: DefineStepRuleJson = formatDefineStepData(mockData);
-      const expected = {
+      const result = formatDefineStepData(mockData);
+      const expected: DefineStepRuleJson = {
```

Comment:
NOTE: These fixes for types are that return types which are explicit from the function don't need to be duplicate typed. However, the expected types should be typed as before they were just objects. This lead to me finding an extra spot we were missing types from our code base I fix below.